### ISSUE TITLE
VuePress v2 please for dark mode

### ISSUE THREAD

**Issue by tomByrer** [theatre-js/theatre-docs#1]
All white screens trigger my concussion.  I use "Dark Mode Reader" for sites that don't support my OS preference, but it would be better if you could switch to [VP v2](https://v2.vuepress.vuejs.org/) for their auto-darkmode please.

**Comment by AriaMinaei** [comment]
I fully agree. I had some issues switching to v2 before, but I hope it's ironed out by now. I will give it another try. Either way, dark mode is a must.
